### ISSUE TITLE
bootstrap issue on Fedora

### ISSUE THREAD

**Issue by przemekklosowski** [gittup/tup#343]
Fedora splits development packages, so when tup requires FUSE to bootstrap, it fails even though the fuse package is installed. ./bootstrap-nofuse.sh does not seem to work; the correct workaround is to install the fuse-devel  package.

**Comment by przemekklosowski** [comment]
I suggest changing the wording to "require the FUSE development package" to help people realize that they need the one with .h files, not just the binaries.

**Comment by kingcocomango** [comment]
This is true on all distros, tup uses fuse so building it requires libfuse

**Comment by shonfeder** [comment]
To be very clear for future seekers, you need to

```
sudo apt install libfuse-dev
```

**Comment by aljoni** [comment]
For Fedora you need to run `sudo dnf install fuse-devel`.

**Comment by AndydeCleyre** [comment]
Copying my comment from #417, now that I see a Fedora-specific issue:

---

Maybe a Fedora user would like to add tup to the [package wishlist](https://fedoraproject.org/wiki/Package_maintainers_wishlist), and/or try contributing a package to the third party [terra repo](https://developer.fyralabs.com/terra/contributing).
